tests/testthat/_snaps/after-wrappers.md

```diff
@@ -36,20 +36,21 @@
       [61] "min"                       "min_horizontal"           
       [63] "n_unique"                  "numeric_dtypes"           
       [65] "options"                   "polars_info"              
-      [67] "raw_list"                  "read_csv"                 
-      [69] "read_ndjson"               "read_parquet"             
-      [71] "reduce"                    "reset_options"            
-      [73] "rolling_corr"              "rolling_cov"              
-      [75] "same_outer_dt"             "scan_csv"                 
-      [77] "scan_ipc"                  "scan_ndjson"              
-      [79] "scan_parquet"              "select"                   
-      [81] "set_global_rpool_cap"      "set_options"              
-      [83] "show_all_public_functions" "show_all_public_methods"  
-      [85] "std"                       "struct"                   
-      [87] "sum"                       "sum_horizontal"           
-      [89] "tail"                      "threadpool_size"          
-      [91] "using_string_cache"        "var"                      
-      [93] "when"                      "with_string_cache"        
+      [67] "pub_class_env"             "pub_env"                  
```

Comment:
I feel that `pub_class_env` and `pub_env` do not need to be made public.
Is it possible to remove the `pl_` prefix from these original names?

Reply:
Done